ghost/email-service/test/email-renderer.test.js

```diff
@@ -1057,6 +1057,40 @@ describe('Email renderer', function () {
             should($('.preheader').text()).eql('Custom excerpt');
         });
 
+        it('does not include member-only content in preheader', async function () {
+            renderedPost = '<div> Lexical Test </div> <div data-gh-segment="status:-free"> members only section</div> some text for both <!--members-only--> finishing part only for members';
```

Comment:
We need to have the same test with paid member only content, but without the paywall `<!--members-only-->`, because in that case it should also hide the `<div data-gh-segment="status:-free">` part from the preheader

Reply:
(you'll probably add some more tests to to get the coverage back to 100%)

Reply:
👍 Updated in https://github.com/TryGhost/Ghost/pull/17137/commits/553fadb1f31277b5f5f92d1318cb9c6663707c4b

Heh the coverage had fallen due to the test modifying the global `renderedPost` var meaning subsequent tests were using this modified version which in turn meant a specific code path never got executed 😳